internal/gitutil/git.go

```diff
@@ -78,6 +79,26 @@ func GetRemoteURL(dir string) (string, error) {
 func Exec(pwd string, args ...string) (string, error) {
 	klog.V(4).Infof("Going to run git %s", strings.Join(args, " "))
 	cmd := osexec.Command("git", args...)
+
+	// NOTE: https://github.com/kubernetes-sigs/krew/pull/739
+	if runtime.GOOS == "windows" {
+		environ := os.Environ()
```

Comment:
do you mind if we move `environ:=...` and `cmd.Env=environ` to outside the `if`?

Reply:
Sure.